### PR TITLE
Revamp the way we retrieve the next event

### PR DESCRIPTION
**Pull request by mxie**
Sometimes our Eventbrite account isn't the sole source of the registration
pages. For instance, we are a subuser under General Assembly's EB account.
Since subusers are not allowed to set up their own API keys, the easiest way to
access an event is to just retrieve the event by its ID (which can be found in
the EB url). This eliminates the need to reach through an organizer and their
events.

### CODE REVIEW

File: .sample.env [hunk 1]
```diff
@@ -1,3 +1,4 @@
 RACK_ENV=development
 SECRET_KEY_BASE=development_secret
 EVENTBRITE_ACCESS_TOKEN=GET_THIS_FROM_HEROKU_CONFIG
+EVENT_ID=GET_FROM_EVENTBRITE
```
Comment (gabebw) [inline .sample.env]: Maybe call this `NEXT_EVENT_ID`?

File: spec/eventbrite/event_finder_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Eventbrite::EventFinder.find("123")
+
+        expect(Eventbrite::Event).to have_received(:new).with(event_body)
+      end
+    end
+
+    context "when there's a bad response" do
+      it "returns a placeholder event" do
+        stub_request(:get, eventbrite_api_url).to_return(
+          body: { id: "123" }.to_json,
```
Comment (gabebw) [inline spec/eventbrite/event_finder_spec.rb]: Should this have `headers: { "Content-Type" => "application/json" }` too?

Maybe it's worth extracting a `stub_json_get_request` like:

```ruby
def stub_json_get_request(body:)
  stub_request(:get, eventbrite_api_url).to_return(
    headers: { "Content-Type" => "application/json" },
    body: body.to_json,
  )
end
```

File: lib/eventbrite/finder.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+require "httparty"
+
+module Eventbrite
+  class Finder
+    include ::HTTParty
```
Comment (gabebw) [inline lib/eventbrite/finder.rb]: Do we need the `::`?

File: .sample.env (hunk 1 above)
Comment (mxie) [inline .sample.env]: I like it!

File: lib/eventbrite/finder.rb (hunk 3 above)
Comment (mxie) [inline lib/eventbrite/finder.rb]: I'm going try it without the `::` again, but tests were complaining about not being able to find it under the `Eventbrite` scope.

**Comment by mxie** [comment]
@gabebw Addressed all your feedback. Ready for re-review!

File: lib/eventbrite.rb [hunk 4]
```diff
@@ -1,62 +0,0 @@
-require "httparty"
```
Comment (gabebw) [inline lib/eventbrite.rb]: Can `httparty` be removed from the Gemfile?

Comment (gabebw) [inline lib/eventbrite.rb]: Ah, nope.

**Comment by gabebw** [comment]
Looks great! :+1:

**Pull request merged**